src/main/java/com/mraof/minestuck/MinestuckConfig.java

```diff
@@ -32,6 +33,11 @@ private Common(ForgeConfigSpec.Builder builder)
 			logItemsWithRecipeAndCost = builder.comment("Makes the recipe-generated grist cost process log any items that has a grist cost, but which could also be provided as a recipe generated cost. Useful for finding items that probably do not need manual grist costs.")
 					.define("logItemsWithRecipeAndCost", false);
 			builder.pop();
+			
+			builder.push("performance");
+			entryDelay = builder.comment("Time reserved for worldgen between generating the land and the actual entry. Measured in ticks. Try increasing this if entry halts the game too much.")
+					.defineInRange("entryDelay", 40, 0, Integer.MAX_VALUE);
```

Comment:
Recommendation to double the default value

Reply:
Actually perhaps even more. That way performance is optimal and players get more of a chance to see the particle effects